tinyDA/chain.py

```diff
@@ -622,7 +623,8 @@ def __init__(
             # it may not be ergodic.
             elif self.adaptive_error_model == "state-dependent":
                 pass
-
+            else:
+                raise NameError("Adaptive error model can only be state-dependent, state-independent or None.")
```

Comment:
Can we make this a `ValueError` too?

Reply:
@louisekluge Did you want to also change this to a `ValueError`?

Reply:
Yes, indeed.